app/src/atoms/buttons/TouchControlButton.tsx

```diff
@@ -0,0 +1,102 @@
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+import {
+  BORDERS,
+  Btn,
+  COLORS,
+  CURSOR_DEFAULT,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+import { getIsOnDevice } from '/app/redux/config'
+
+import type { MouseEventHandler } from 'react'
+
+interface TouchControlProps {
+  text1: string
+  isActive: boolean
+  onClick?: MouseEventHandler
```

Comment:
```suggestion
  onClick: () => void
```
Since this is a button, the `onClick` should be required. Because we are passing in custom JS for the `onClick`, let's make the return type `() => void`.